### ISSUE TITLE
There is any way i can get unban?

### ISSUE THREAD

**Issue by JordiSins** [ObliqueNET/Stratos#252]
I got banned for x-ray but im sorry and i wont cheat again. Can i please get unbanned since i wont cheat again.
Btw i was using the x-ray for quartz not for dungeons.

**Comment by MrBritish8** [comment]
Yes you would need to provide proof you no longer have the x-ray then you would be able to play again but this comes at a cost if your caught again you will be banned with no chance of appeal

**Comment by JordiSins** [comment]
![image](https://user-images.githubusercontent.com/51290623/58885685-8db5b880-86da-11e9-9dcd-bcaa9cdf4edd.png)

**Comment by JordiSins** [comment]
i remove it when i got banned and i wont use it again

**Comment by JordiSins** [comment]
thank you kind sir, i wont use it again
